<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="createTemperatureButton">
     <property name="text">
      <string>Добавить датчик температуры</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="createLightInfoButton">
     <property name="text">
      <string>Активировать датчик света</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="createLeakInfoButton">
     <property name="text">
      <string>Добавить датчик протечки</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="createDateTimeInfoButton">
     <property name="text">
      <string>Добавить Дата время</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
